recipes/temporalio/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "temporalio" %}
+{% set version = "1.14.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/temporalio-{{ version }}.tar.gz
+  sha256: b240cf56f64add65beb75bd18aa854ac35bdc2505097af5af1e235d611190a9d
+
+build:
+  skip: true  # [win or py<39]
```

```suggestion
  skip: true  # [win]
```
We don't support Python 3.8 anymore, so we only need to skip Windows.